### DISCUSSION TITLE
Kafka Connecter, Serialiser/Deserialiser Avro Format

### DISCUSSION THREAD

**Discussion by ch14ern** [apache/streampipes#2950] [Ideas]
Hello,

I am currently experiencing that with Kafka it is not possible to deserialise messages in Avro format.
Is it possible to include this in the next release when the message is in avro format, give the user a field to paste in the schema registry url and then streampipes can also receive and deserialise avro format.

**Comment by dominikriemer** [comment]
Hi @ch14ern 
do you already have a solution in mind? E.g., Avro is available as an additional format in the Kafka Adapter format selection panel and once the option is selected, users can provide the schema registry URL? Are there any other schema registry options necessary, e.g., authentication options?
